{{cookiecutter.project_slug}}/Pipfile

```diff
@@ -11,7 +11,7 @@ python_version = "3.7"
 [packages]
 django = "~=2.2.3"
 django-environ = "~=0.4.5"
-psycopg2 = "~=2.8.5"
+psycopg2-binary = "~=2.8.5"
```

Comment:
While using the binary will speed up builds it's not recommended for production: https://www.psycopg.org/docs/install.html#binary-install-from-pypi